.github/workflows/ci.yml

```diff
@@ -194,6 +194,12 @@ jobs:
           cat >> ~/.bash_profile << ___
             cd "$(pwd)"
 
+            # Configure canary build.
+            if [ "$DENO_CANARY" == "true" ]
+            then
+              export DENO_CANARY=true
```

Comment:
environment variables in github actions are not persisted across steps by default. you'll have to use a special environment variable `GITHUB_ENV`

```sh
echo "DENO_CANARY=true" >> $GITHUB_ENV
```

more info: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable